generate.sh

```diff
@@ -18,6 +18,13 @@ if [ -e "$SUMMARY_FILEPATH" ]; then
     rm "$SUMMARY_FILEPATH"
 fi
 
+
+if [ -z  `which log2dch` ]; then
+  echo "Missing log2dch script from https://github.com/CanonicalLtd/uss-tableflip for cloud-init and curtin dev-summary"
+  exit 1
```

Comment:
Does this mean I need to have uss-tableflip/scripts in my path at all times?

Reply:
yeah, at least it's needed to generate the dynamic content from the git commit messages.  Do you want it to be optional? we can do that too.